pkg/pluginx/command.go

```diff
@@ -0,0 +1,118 @@
+package pluginx
```

Comment:
General comment: not sure what about error wrapping (if we used Go-native wrapping, then it would be easy to get the root error without an external library), personally I would add it at some point. But I think it's fine how it is for now 👍